.travis.yml

```diff
@@ -1,9 +1,11 @@
 language: node_js
 node_js:
+  - '6.1'
+  - '5.5'
+  - '4.2'
   - 'iojs-1.2'
   - 'iojs-1.1'
   - 'iojs-1.0'
   - '0.12'
-  - '0.11'
```

Comment:
Failing with `0.11` due error in dependencies. I think `0.11` support can be removed but final decision is up to you guys. + tests against recent versions of node added. This is all from my side. Feel free to ping me if you I can help with something and thank you for maintaining this package!